AnwesenheitsSimulation/module.php

```diff
@@ -25,6 +25,7 @@ public function Create()
 
         //Variables
         $this->RegisterVariableString('SimulationView', $this->Translate('Simulation preview'), '~HTMLBox');
+        $this->RegisterVariableInteger('NextTrigger', $this->Translate('Next Trigger'), '');
```

Comment:
The last parameter (profile) should be ~UnixTimestamp. (See here: https://www.symcon.de/en/service/documentation/components/visualizations/object-presentation/datetime/)